dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java

```diff
@@ -67,15 +71,18 @@ public static Session createConnection(final NewSessionModel newSessionModel, fi
                     LOG.error("Invalid Access ID or Secret Key: ", e);
                     alert.showAlert(resourceBundle.getString("invalidIDKEYMessage"));
                 }
+            } else if (e.getStatusCode() == 301) {
+                LOG.error("BlackPearl return an unexpected status code, indicating we are attempting to make a data path request on the Management port: ", e);
+                alert.showAlert(resourceBundle.getString("invalidEndpointMessage"));
             } else {
-                LOG.error("BlackPearl return an unexpected status code we did not expect: {}", e);
+                LOG.error("BlackPearl return an unexpected status code: ", e);
                 alert.showAlert(resourceBundle.getString("unexpectedStatusMessage"));
             }
         } catch (final IOException ioe) {
             LOG.error("Encountered a networking error: ", ioe);
             alert.showAlert(resourceBundle.getString("networkErrorMessage"));
         } catch (final RuntimeException rte) {
-            LOG.error("Something went wrong.", rte);
+            LOG.error("Something went wrong: ", rte);
```

Comment:
remove the punctuation in the log message.  The logger already has some when it prints out the message